doc/rules/control_structure/trailing_comma_in_multiline.rst

```diff
@@ -23,7 +23,7 @@ Default value: ``false``
 Where to fix multiline trailing comma (PHP >= 8.0 for ``parameters`` and
 ``match``).
 
-Allowed values: a subset of ``['arrays', 'arguments', 'parameters', 'match']``
+Allowed values: a subset of ``['arguments', 'arrays', 'array_destructuring', 'match', 'parameters']``
```

Comment:
Is there a need for a separate option? I think it would make sense to fix array destructuring when the `arrays` option is enabled.

Reply:
It is mainly for BC reasons, now the new feature of fixing array destructing is opt-in. We can always consider to merge two functionalities under one config option in the next major release.

Reply:
We have BC breaks like this every release, e.g. we've just merged #6583. BTW, do we have a place to keep track of things we want to consider before the next major?

Reply:
>do we have a place to keep track of things we want to consider before the next major?

not that I know of

> We have BC breaks like this every release, e.g. we've just merged https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/6583.

I don't think there is a BC break in that PR

On the topic of this PR, what is the problem with the current proposal?

Reply:
> I don't think there is a BC break in that PR

Neither do I, some fixer just learned a new trick, just like in this one :) I fail to see how they're different.

Both for users and for us it's easier not to include an option if it's unnecessary than to remove it in the next major.

I do not want to block this PR, if you feel that the BC break would be too significant feel free to ignore my comments :)